xml/System.ComponentModel/DateOnlyConverter.xml

```diff
@@ -91,7 +91,7 @@
       <Docs>
         <param name="context">To be added.</param>
         <param name="destinationType">To be added.</param>
-        <summary>To be added.</summary>
+        <summary>Gets a value indicating whether this converter can convert an object to the given destination type using the context.</summary>
         <returns>To be added.</returns>
```

Comment:
```suggestion
        <returns><see langword="true" /> if this converter can perform the conversion; otherwise, <see langword="false" />.</returns>
```